views/update/remove-is-entity-registered-owner.html

```diff
@@ -40,17 +40,19 @@ <h1 class="govuk-heading-xl govuk-!-margin-bottom-3">Is the overseas entity curr
           },
           items: [
             {
-              value: "yes",
+              value: 1,
```

Comment:
If changing from a string to boolean can we use true/false values for readability?

Reply:
i think 1/0 might be the nunjucks component defaults, i know we used 1/0 in registration so they are consistent